trino/README.md

```diff
@@ -0,0 +1,59 @@
+# Agent Check: Trino
+
+## Overview
+
+This check collects [Trino][1] metrics, such as the following examples:
+
+- Overall activity metrics: completed/failed queries, data input/output size, execution time.
+- Performance metrics: cluster memory, input CPU, execution CPU time.
+
+## Setup
+
+### Installation
+
+The Trino check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `trino.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory, to start collecting your Trino performance data.
+   See the [sample trino.d/conf.yaml][3] for all available configuration options.
+
+   This check has a limit of 350 metrics per instance. The number of returned metrics is indicated when running the Datadog Agent [status command][4].
+   You can specify the metrics you are interested in by editing the [configuration][3].
+   To learn how to customize the metrics to collect, read  [JMX Checks][5].
+   If you need to monitor more metrics, contact [Datadog support][6].
+
+2. [Restart the Agent][7]
+
+### Validation
+
+[Run the Agent's `status` subcommand][4] and look for Trino under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+Trino does not include any metrics.
```

Comment:
Can you update this section to link to the `metadata.csv` since it does collect metrics?